laundry/views.py

```diff
@@ -0,0 +1,124 @@
+import calendar
+import datetime
+
+from django.utils import timezone
+from django.utils.timezone import make_aware
+from requests.exceptions import HTTPError
+from rest_framework import generics
+from rest_framework.response import Response
+
+from laundry.api_wrapper import Laundry
+from laundry.models import LaundrySnapshot
+from laundry.serializers import LaundrySnapshotSerializer
+
+
+laundry = Laundry()
+
+
+# should i just use the regular API view with a GET header?
+class Halls(generics.ListAPIView):
```

Comment:
I think for all of these you should be fine using just APIView and a `get` method. I think you could also use JsonResponse here instead of Response

Reply:
is there a particular difference between jsonresponse and response other than response being a DRF thing?

Reply:
Actually, I take that back. JsonResponse is a [native django class](https://docs.djangoproject.com/en/3.1/ref/request-response/#jsonresponse-objects) not a DRF one, and it doesn't have the ability to negotiate a renderer. Response is the way to go!